Makefile

```diff
@@ -11,6 +11,8 @@ TORONTO_BIDS_SERVER_UBERJAR = $(TORONTO_BIDS_SERVER_TARGET_DIR)/uberjar/$(TORONT
 TORONTO_BIDS_SERVER_UBERJAR_STANDALONE = $(TORONTO_BIDS_SERVER_TARGET_DIR)/uberjar/$(TORONTO_BIDS_SERVER_REVISION)-standalone.jar
 
 setup-py: $(VENV)
+$(PYTHON): setup-py
+$(PIP): setup-py
```

Comment:
I do not believe these are necessary? They would only need to be here if `$(PYTHON)` or `$(PIP)` was listed as a dependency under other targets.

Reply:
yupp, totally we're not building them, they would be covered by `VENV` , updated